Answers.md

```diff
@@ -2,10 +2,34 @@
 
 1.  What is React JS and what problems does it try and solve?
 
+In my mind, React JS does two big things. First, it allows developers to easily create reuseable components. When building a social-media website, for instance, there are potentially thousands of posts or tweets that need to be rendered. Doing that manually is impossible, so reuseability is key. And, thank goodness, React makes life easy by using JSX, which allows developers to write components with a mix of HTML and JavaScript. That means developers don't have to write countless `document.createElement()`s along the way, *plus* things can be dynamic by using JS with its `ternary operator` and other useful things to make the HTML exactly what's desired.
+
+The second big thing React JS brings to the table is, well, reactivity. As data changes or users interact with the web page, React allows the page to dynamically change as well. No need to refresh the page. It's *alive*.
+
 1.  What does it mean to _think_ in react?
 
+I don't know... My best answer to what I think this question is asking is that:
+
+React is built both with a distinct top-down heirarchy and with components. This impacts the way a React developer should think about approaching a project. 
+
+With components, the developer should endeavor to make each component do a single thing. Then, that component (and, likely, other components, too) is used by another component to do a bigger thing in concert. That means the developer should break down complex UIs into small parts.
+
+With top-down heirarchy, I am referring to the behind-the-scenes stuff that makes the reactivity of React work. In most cases, data and information is both retrieved and stored at a higher level than the bottom-level components. That data and information is then distributed to those lower-level components, and cannot be directly sent back up the chain (there are ways, of course, but it typically involves sending a function *from* the higher level along with the data), so React developers must be mindful of the heirarchy.
+

```

Comment:
You said you don't know here, but I think this is a great answer! I think components or boxes that create an whole of what we are creating. Love your answer.